Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,39 @@
-﻿namespace Logger;
+using System;
+namespace Logger;
 
 public static class BaseLoggerMixins
 {
-    
+    public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+    {
+        if (baseLogger is null) { throw new ArgumentNullException(); }
```

Comment:
```suggestion
        if (baseLogger is null) 
        { 
            throw new ArgumentNullException(); 
        }
```
OR 
```suggestion
        if (baseLogger is null) throw new ArgumentNullException();
```
Although the second suggestion has some strong opinions around not doing that, the first suggestion is the safer bet

Reply:
If I have the space, I will prefer to put it on one line. But I always use braces to remain syntactically consistent.